sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/IntervalExpressionsSuite.scala

```diff
@@ -197,10 +198,22 @@ class IntervalExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("multiply") {
-    def check(interval: String, num: Double, expected: String): Unit = {
-      checkEvaluation(
-        MultiplyInterval(Literal(stringToInterval(interval)), Literal(num)),
-        if (expected == null) null else stringToInterval(expected))
+    def check(
+        interval: String,
+        num: Double,
+        expected: String,
+        checkException: Boolean = false): Unit = {
+      Seq("true", "false").foreach { v =>
+        withSQLConf(SQLConf.ANSI_ENABLED.key -> v) {
+          if (checkException) {
+            checkExceptionInExpression[ArithmeticException](
+              MultiplyInterval(Literal(stringToInterval(interval)), Literal(num)), expected)
```

Comment:
we can add a `val expr = MultiplyInterval(Literal(stringToInterval(interval)), Literal(num))` at beginning, to save code duplication.

Reply:
yes, we can also use `safeStringToInterval(expected) == null` to choose way to check too